src/frontend/apps/desk/src/pages/teams/create.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Input, Loader } from '@openfun/cunningham-react';
+import { ReactElement, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import IconGroup from '@/assets/icons/icon-group2.svg';
+import { Box, StyledLink, Text, WhiteBox } from '@/components';
+import { useCunninghamTheme } from '@/cunningham';
+import { useCreateTeam } from '@/features/teams';
+import { NextPageWithLayout } from '@/types/next';
+
+import TeamLayout from './TeamLayout';
+
+const Page: NextPageWithLayout = () => {
```

Comment:
do you plan to support unique team name in the next PR?

Reply:
I mean the form error related to the fact the name submitted is not unique

Reply:
Yes, I have a PR (https://github.com/numerique-gouv/people/pull/52) managing the errors.